python/cudf_polars/cudf_polars/experimental/benchmarks/pdsh.py

```diff
@@ -1072,6 +1072,13 @@ def _query_type(query: int | str) -> list[int]:
     action="store_true",
     help="Debug run.",
 )
+parser.add_argument(
+    "--ucx",
```

Comment:
Maybe?
```suggestion
    "--ucx-frontend",
```

Reply:
Thanks Matt. I used @pentschev suggestion of protocol

Reply:
Thanks Matthew, I overruled your comment with mine because the proper term should be "protocol" for Dask, but that was a good trigger for the conversation which I initially overlooked. 😄